cmd/root.go

```diff
@@ -92,6 +108,43 @@ func initConfig() {
 	}
 }
 
+func downloadConfig(cfgURL string) (string, error) {
+	fileURL, err := url.Parse(cfgURL)
+	if err != nil {
+		return "", err
+	}
+	path := fileURL.Path
+	segments := strings.Split(path, "/")
+	fileName := segments[len(segments)-1]
+
+	file, err := ioutil.TempFile("", fmt.Sprintf("*-%s", fileName))
```

Comment:
Neat, I didn't know this was a thing

Reply:
Just discovered it recently. Used it in the new Reckoner code. It's pretty convenient.